snippets/app-router/catch-all.mdx

```diff
@@ -1,12 +1,10 @@
-```tsx src/app/[[...path]]/page.tsx
+```tsx src/app/[[...path]]/page.tsx (Next.js 15)
 import { getSiteVersion } from "@makeswift/runtime/next/server";
```

Comment:
Note, the API is `siteVersion`, not `getSiteVersion`!

I personally like code snippets that can run, or at the very least are checked. Hopefully we can do something like that once we move away from Mintlify. I have some thoughts on some tools that can help us with that.